airflow/models/mappedoperator.py

```diff
@@ -162,6 +165,7 @@ class OperatorPartial:
     """
 
     operator_class: Type["BaseOperator"]
+    params: Union[ParamsDict, dict]
```

Comment:
To be able to expand a task against `params`, we need to be able to “merge” default DAG- and task-level params with the user-mapped params. So `params` is split out of other partial kwargs and treated specially. Partial kwargs should _never_ contain the `"params"` key.